### ISSUE TITLE
Using insertCardAtIndexRange without adding more elements to datasource

### ISSUE THREAD

**Issue by Daemon-Devarshi** [Yalantis/Koloda#196]
I am trying to implement a kind of continuous card viewer such that if user swipes out the first card it is appended at the end of the stack.

I am able to achieve this behavior by using below code:

```
func koloda(koloda: KolodaView, didSwipeCardAtIndex index: UInt, inDirection direction: SwipeResultDirection) {
        if let aTeam = teams?[Int(index)] {
            teams!.append(aTeam)
            let position = koloda.countOfCards - 1
            koloda.insertCardAtIndexRange(position...position)
        }
    }
```

It is behaving as expected but only problem with this code is - 

> On each swipe out it is adding a new element in teams, which is my data source array, so in case of first card swipe out there will be a redundant team object added at the end of data source array.

Is there any way I can make it continuous (or circular) without increasing the number of elements in data source? Any code reference or pointer as clue would be of great help.

**Comment by NSMyself** [comment]
I just cloned the current version of this lib. 
Doesn't it already have this behaviour? If I swipe until I pass Sheldon Cooper (the last item), Hugh Laurie (the first item) appears again.

**Comment by muiruri** [comment]
@Daemon-Devarshi have you tried `resetCurrentCardIndex()` function? I have used it and it brings up the first card after the last one is swiped.

**Comment by M-I-N** [comment]
Implemented via #357
